storage/stiface/stiface_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	"os"
 	"testing"
 
-	"golang.org/x/net/context"
+	"context"
```

Comment:
Could you move this up? Ditto elsewhere.

Reply:
Done. Forgot that `goimports` doesn't mess with groups.